pkg/backup/show.go

```diff
@@ -225,45 +222,6 @@ func showBackupPlanHook(
 	}
 	exprEval := p.ExprEvaluator("SHOW BACKUP")
 
-	// TODO(dt): find move this to its own hook.
```

Comment:
So we're ripping out `SHOW BACKUP CONNECTION` here, rather than pointing it at the moved cloud check code?

I suppose this is okay since we never doc'ed it, but I think if we're doing that we can delete the case above about its header and we can delete the `BackupConnectionTest` parse/AST support too, right?

Reply:
Yeah since it uses the `planHookFn` and the moved code creates a standalone `planNode`, trying to just have it point to it I think is more effort than it is worth. I'll remove all the old code in a follow up.